packages/aws-cdk-lib/aws-cloudwatch-actions/lib/lambda.ts

```diff
@@ -21,7 +22,8 @@ export class LambdaAction implements cloudwatch.IAlarmAction {
    * @see https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_PutMetricAlarm.html
    */
   bind(_scope: Construct, _alarm: cloudwatch.IAlarm): cloudwatch.AlarmActionConfig {
```

Comment:
nit: In general, we add `_` to parameter to indicate it's unused. Since we're using it in `idPrefix`, it would be nice to remove the `_`. (I know it's not from you, but would be nice to make the style correct).